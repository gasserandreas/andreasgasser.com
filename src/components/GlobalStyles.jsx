import { createGlobalStyle } from 'styled-components';
import reset from 'styled-reset-advanced';

/**
 * Backup old GlobalStyles
 */
// * {
//   box-sizing: border-box;
//   outline: 0;
// }

// html,
// body,
// #___gatsby,
// #___gatsby > div {
//   margin: 0;
//   padding: 0;
//   min-height: 100%;
//   width: 100%;
//   height: 100%;
//   box-sizing: border-box;
// }

// body {
//   margin: 0;
//   padding: 0;
//   font-family: sans-serif;
// }

const GlobalStyle = createGlobalStyle`
  ${reset};
 
  body {
    font-family: 'Helvetica Neue', 'Helvetica', sans-serif;
    font-weight: 500;
  }
`;

export default GlobalStyle;
