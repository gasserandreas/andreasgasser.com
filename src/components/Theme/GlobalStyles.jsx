import { createGlobalStyle } from 'styled-components';


const GlobalStyle = createGlobalStyle`
  /* get Google font */
  @import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');

  * {
    box-sizing: border-box;
    outline: 0;
  }

  html,
  body,
  #___gatsby,
  #___gatsby > div {
    margin: 0;
    padding: 0;
    min-height: 100%;
    width: 100%;
    height: 100%;
    box-sizing: border-box;
  }

  body {
    margin: 0;
    padding: 0;
    font-family: 'Open Sans', 'Helvetica Neue', Helvetica, sans-serif;
  }
`;

export default GlobalStyle;
