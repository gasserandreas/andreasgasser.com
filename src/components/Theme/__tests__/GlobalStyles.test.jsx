import GlobalStyles from '../GlobalStyles';

describe('components/Theme/GlobalStyles', () => {
  it('should provide GlobalStyles', () => {
    expect(GlobalStyles).toMatchSnapshot();
  });
});
