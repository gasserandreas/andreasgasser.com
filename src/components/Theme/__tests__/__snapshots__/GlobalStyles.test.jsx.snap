// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Theme/GlobalStyles should provide GlobalStyles 1`] = `[Function]`;
